CMakeLists.txt

```diff
@@ -397,6 +406,17 @@ if(GAMMARAY_BUILD_DOCS)
   find_package(Qt${Qt_VERSION_MAJOR} NO_MODULE QUIET OPTIONAL_COMPONENTS Help LinguistTools)
 endif()
 
+if(${Qt_VERSION_MAJOR} GREATER_EQUAL 6)
+  set(CMAKE_CXX_STANDARD 17)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  add_definitions(-DGAMMARAY_QT6_TODO)
```

Comment:
Do we still need this? That was supposed to be just for temporary scaffolding during porting.